test/recognize_test.rb

```diff
@@ -269,4 +269,21 @@
       end
     end
   end
+
+  describe '#recognize without routes' do
```

Comment:
this example description to me is duplication of line 279, there should be only 1